[2023-03-02T19:02:33.079+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: metadata_scraper.fetch_nexrad_stations manual__2023-03-02T19:02:29.662309+00:00 [queued]>
[2023-03-02T19:02:33.100+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: metadata_scraper.fetch_nexrad_stations manual__2023-03-02T19:02:29.662309+00:00 [queued]>
[2023-03-02T19:02:33.102+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-02T19:02:33.103+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-03-02T19:02:33.108+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-02T19:02:33.126+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): fetch_nexrad_stations> on 2023-03-02 19:02:29.662309+00:00
[2023-03-02T19:02:33.151+0000] {standard_task_runner.py:55} INFO - Started process 2299 to run task
[2023-03-02T19:02:33.162+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'metadata_scraper', 'fetch_nexrad_stations', 'manual__2023-03-02T19:02:29.662309+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/scrapper.py', '--cfg-path', '/tmp/tmpq1iztop2']
[2023-03-02T19:02:33.170+0000] {standard_task_runner.py:83} INFO - Job 27: Subtask fetch_nexrad_stations
[2023-03-02T19:02:33.335+0000] {task_command.py:388} INFO - Running <TaskInstance: metadata_scraper.fetch_nexrad_stations manual__2023-03-02T19:02:29.662309+00:00 [running]> on host f6e174e28dc0
[2023-03-02T19:02:33.531+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Charu Singh
AIRFLOW_CTX_DAG_ID=metadata_scraper
AIRFLOW_CTX_TASK_ID=fetch_nexrad_stations
AIRFLOW_CTX_EXECUTION_DATE=2023-03-02T19:02:29.662309+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-02T19:02:29.662309+00:00
[2023-03-02T19:02:33.921+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/scrapper.py", line 170, in nexrad_stations
    df = pd.read_csv("../working_dir/data/nexrad-stations.txt", colspecs=cols, skiprows=[1])
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
TypeError: read_csv() got an unexpected keyword argument 'colspecs'
[2023-03-02T19:02:34.132+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=metadata_scraper, task_id=fetch_nexrad_stations, execution_date=20230302T190229, start_date=20230302T190233, end_date=20230302T190234
[2023-03-02T19:02:34.317+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 27 for task fetch_nexrad_stations (read_csv() got an unexpected keyword argument 'colspecs'; 2299)
[2023-03-02T19:02:34.431+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-03-02T19:02:34.651+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
